crates/core/src/factories/core_fetch/test.rs

```diff
@@ -4,15 +4,19 @@ use std::{
 };
 
 use bytes::Bytes;
+
 use kitsune2_api::{
     fetch::{deserialize_op_ids, Fetch},
     id::Id,
     transport::Transport,
     AgentId, K2Error, OpId, SpaceId, Url,
 };
+
+use kitsune2_test_utils::agent::*;
```

Comment:
Why spaces between imports? Can keep them in a block and let `cargo fmt` keep them tidy?

Reply:
I touched this code, I can fix it : ) -- https://github.com/holochain/kitsune2/pull/55/commits/db795946a666d7f60689ebd293f007589e538734